src/App.js

```diff
@@ -120,6 +120,7 @@ class OldApp extends Component {
     this.setState({ eventSubscription: eventSub });
     appEvents.on('transactionConfirmed', this.handleTransactionConfirmed);
 
+    await analyticsV2.initializeRudderstack();
     await this.setupDeeplinking();
```

Comment:
I think we should execute these in parallel unless they depend on one another. I also get a little nervous throwing stuff in the initialization phase of the app that isn't technically under our control. (e.g.- If Rudderstack were to have an outage, does that stall our app from initializing?)